diff --git a/CMakeLists.txt b/CMakeLists.txt
index e882fc3..12c526b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -73,9 +73,11 @@ target_link_libraries(main PRIVATE
     ${MEDIAPLATFORM_LIB}
 )
 
+target_link_libraries(main PRIVATE ${LOG_SHARED_LIB})
+
 get_target_property(MAIN_COMPILE_DEFINITIONS main COMPILE_DEFINITIONS)
 if(NOT "MyRelease" IN_LIST MAIN_COMPILE_DEFINITIONS)
-    target_link_libraries(main PRIVATE ${LOG_SHARED_LIB} ${CURL_SHARED_LIB})
+    target_link_libraries(main PRIVATE ${CURL_SHARED_LIB})
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Werror -g")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Werror -g")
 else()
